docs/napari-plugin-guide.md

```diff
@@ -12,145 +11,79 @@ Launch `napari` with `recOrder`
 napari -w recOrder-napari
 ```
 
-## `recOrder` Calibration
-
-The first step in the QLIPP process is to calibrate the universal polarizer. This process involves generating the polarization states and acquiring a background **on an empty FOV**.  The light path must also be in **Kohler Illumination** in order to ensure uniform illumination of the sample. Steps on Kohler illumination can be found here https://www.microscopyu.com/tutorials/kohler.
-
-More info on the details of the calibration process and use of the plugin can be found on Zenodo which were taken from a workshop at the Chan-Zuckerberg Biohub in June 2021.
-
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5178487.svg)](https://doi.org/10.5281/zenodo.5178487)
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5135889.svg)](https://doi.org/10.5281/zenodo.5135889)
-
-Turn on online mode by clicking "Switch To Online" button.
+## Calibration tab
+The first step in the QLIPP process is to calibrate the liquid crystals. This process involves generating the polarization states and acquiring a background **on an empty FOV**, so begin by placing your sample on the stage and focussing on the surface of the coverslip or well. 
```

Comment:
Not an error, but ['focusing'](https://books.google.com/ngrams/graph?content=focusing%2C+focussing&year_start=1900&year_end=2019&corpus=26&smoothing=3) seems to be more common.

Reply:
imma commonwealth boy!!!